x/crosschain/keeper/process_inbound_test.go

```diff
@@ -296,9 +332,9 @@ func TestKeeper_ProcessCrosschainMsgPassing(t *testing.T) {
 		// mock successful UpdateNonce
 		updatedNonce := keepertest.MockUpdateNonce(observerMock, *receiverChain)
 
-		// call ProcessCrosschainMsgPassing
+		// call ProcessInbound
 		cctx := GetERC20Cctx(t, receiver, *receiverChain, "", amount)
-		k.ProcessCrosschainMsgPassing(ctx, cctx)
+		k.ProcessInbound(ctx, cctx)
```

Comment:
Is there a reason we need to change this  ? 
ProcessInbound is just a dispatch function; we can probably create a separate unit test for it, if needed , to verify it calls the correct function based on the condition.

Reply:
Wondering if you wanted to change it back to `ProcessCrosschainMsgPassing` instead of `ProcessInbound` for this , and the other functions in this test ?